modules/Xsens_module/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ set(${EXE_TARGET_NAME}_HDR
 include_directories(
   ${CMAKE_CURRENT_SOURCE_DIR}/include
 )
+find_package(HumanDynamicsEstimation)
```

Comment:
This is already done in https://github.com/robotology/walking-teleoperation/blob/257b8d9cdfb970f57a4abe1195e0bee16d7550d6/cmake/WalkingTeleoperationFindDependencies.cmake#L120 .

Reply:
Thank you.